src/Guard/UserProviderInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace BoShurik\TelegramBotBundle\Guard;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+
+interface UserProviderInterface
```

Comment:
What about separating this interface on `UserProviderInterface` with `loadUserByTelegramId(int $id): ?UserInterface` and `UserFactoryInterface` with `createFromTelegram(array $data): UserInterface`?
With this we can make creating users optional (only when optional `UserFactoryInterface` passed to constructor)

Reply:
Done! Splitted into `UserFactoryInterface` and `UserLoaderInterface` to avoid name conflicts with the Symfony's `UserProviderInterface`